.github/workflows/aws-minimal-schema.yml

```diff
@@ -10,6 +10,7 @@ on:
 
 jobs:
   check_minimal_schema:
+    runs-on: ubuntu-latest
```

Comment:
This passes now. I wonder though, this is not part of required checks I think. SO it it fails it will display in red but we can miss it. Are there any quick fixes? 

@flostadler you also thought this should run on release?

Reply:
We should be able to add it to the required checks, right?

I think we should add it to the master workflow as well to guard against faulty merges (e.g. when conflict resolution goes rogue).

Reply:
Added to master and releases. I am confused if we manage branch protection through UI or centralized stack in ci-mgmt, but I'll ask. We should be able to add later on to branch protections.